data/uistrings.yaml

```diff
@@ -0,0 +1,143 @@
+# This file is used to generate /uistrings.json
+#
+# To include these strings in content, use:
+# {{% data-include uistrings.<KEY> %}}
+
+composeBuild:
+  Build is an optional part of the Compose Specification. It tells Compose how
+  to (re)build an application from source and lets you define the build process
+  within a Compose file in a portable way.  `build` can be either specified as
+  a single string defining a context path, or as a detailed build definition.
+composeConfigs: 
+  Configs let services to adapt their behaviour without the need to rebuild a
+  Docker image. As with volumes, configs are mounted as files into a
+  container's filesystem. The location of the mount point within the container
+  defaults to `/<config-name>` in Linux containers and `C:\<config-name>` in
+  Windows containers. 
+composeConfigureWatch: 
+  1. Add `watch` sections to one or more services in `compose.yaml`.
+  2. Run `docker compose up --watch` to build and launch a Compose project and start the file watch mode.
+  3. Edit service source files using your preferred IDE or editor.
+composeDeploy:
+  Deploy is an optional part of the Compose Specification. It is used to
+  configure how services are deployed and managed in a Docker Swarm mode.
+  Essentially, it provides a set of deployment specifications for managing the
+  behavior of containers across different environments.
+composeExtension:
+  As with [Fragments](10-fragments.md), Extensions can be used to make your
+  Compose file more efficient and easier to maintain. Extensions can also be
+  used with [anchors and aliases](10-fragments.md).
```

Comment:
I asked Allie to remove the file links from the includes .md. [This was one of them](https://github.com/docker/docs/pull/20080/files#diff-6ccf632e14fb20514a10fd13d998a5526966d7e36be051977d1def44aaf656d1R9)